config/cache/info.js

```diff
@@ -1,10 +1,10 @@
 const Const = require('../../app/common/Const');
-
+const WEEK_IN_MILISECONDS = Const.MINUTE_IN_MILLISECONDS * 60 * 24 * 7
 module.exports = {
   "CurrenciesAllRates": {
     key: "all-rates",
-    TTL: {ttl: 63 * Const.MINUTE_IN_MILLISECONDS},
-    TTLTool: {ttl: 15 * Const.MINUTE_IN_MILLISECONDS}
+    TTL: {ttl: WEEK_IN_MILISECONDS},
```

Comment:
In theory a result whether from blockchain or from Database should only be avail for 1 hour. Now we're not concerning the DB query with a user request anymore (we return nil if cache doesn't hit), hence changing this number to 1 WEEK does not make much sense. I think 60 Minutes is ok.

Reply:
okey, will change to 1 hour